public/app/features/alerting/unified/components/DynamicTable.tsx

```diff
@@ -197,7 +197,6 @@ const getStyles = <T extends unknown>(
     `,
     bodyCell: css`
       overflow: hidden;
-      word-break: break-all;
```

Comment:
I removed this, as mentioned in my review on #48882, because it was breaking mid-word in the table when it doesn't need to.